docs/self-hosting.md

```diff
@@ -0,0 +1,34 @@
+# Self-Hosting Charm
+
+Charm libraries point at our Charmbracelet, Inc. servers by default (that’s cloud.charm.sh), however it's very easy for users to host their own Charm instances. The charm binary is a single, statically-linked executable capable of serving an entire Charm instance. 
+
+To start your charm server, run
+```bash
+charm serve
+```
+in a dedicated terminal window. Then, change the default host by adding `CHARM_HOST=localhost` or `CHARM_HOST=burrito.example.com` to your PATH. 
```

Comment:
You could mention using docker as an option here. https://github.com/charmbracelet/charm/blob/main/docker.md other than that lgtm.

Reply:
Changed in the latest commit :)